test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -3063,23 +2977,14 @@ def test_cgroup_msmom_resize_fail(self):
         # Check that job got requeued
         self.server.log_match("Job;%s;Job requeued" % (jid), starttime=stime)
 
-    @requirements(num_moms=2)
+    @requirements(num_moms=3)
     def test_cgroup_msmom_nodes_only(self):
         """
         Verify that exec_vnode values are trimmed
         when execjob_launch hook prunes job via release_nodes(),
         job is using only vnodes from mother superior host,
         tolerate_node_failures=job_start
         """
-        # Test requires 2 nodes
-        if len(self.moms) < 2:
-            self.skipTest('Test requires at least two Moms as input, '
-                          'use -p moms=<mom1:mom2>')
-        self.load_config(self.cfg7)
```

Comment:
Same here too, I don't you mean to remove the load_config() call.

Reply:
Fixed as well.